cabal.project

```diff
@@ -1,4 +1,4 @@
-index-state: 2021-02-15T00:00:00Z
+index-state: 2021-08-17T00:00:00Z
```

Comment:
The new `cardano-ledger-specs` commit you're pointing to (4595b73a693d0a78f05ecbcbf631caa568dde5d8) uses `index-state: 2021-12-03T06:07:23Z`.

My intuition is that we'll benefit from having the same index-state as our primary upstream library. Would that work here?

Reply:
I switched to `2021-12-03T06:07:23Z` and the build looks fine.

Reply:
Okay, I switched it back.  I remember why now.  The way `nix` is set up, I cannot push it beyond this date.